make/Makefile.common

```diff
@@ -49,13 +49,13 @@ endif # !os_name
 
 cfg ?= release
 
-device_type ?= GPU
+device_type ?= level_zero:gpu
```

Comment:
Let's revert it back to GPU:

```suggestion
device_type ?= GPU
```

It works in conjunciton with the following suggestion.